administrator/components/com_admin/script.php

```diff
@@ -614,6 +614,7 @@ public function deleteUnexistingFiles($dryRun = false, $suppressOutput = false)
             '/libraries/vendor/web-auth/metadata-service/src/Version.php',
             '/libraries/vendor/web-auth/webauthn-lib/src/Server.php',
             '/libraries/vendor/web-token/jwt-signature-algorithm-rsa/RSA.php',
+            '/media/plg_editors_tinymce/js/plugins/highlighter/source.html',
```

Comment:
```suggestion
            '/media/plg_editors_tinymce/js/plugins/highlighter/source.css',
            '/media/plg_editors_tinymce/js/plugins/highlighter/source.html',
            '/media/plg_editors_tinymce/js/plugins/highlighter/source.min.css',
            '/media/plg_editors_tinymce/js/plugins/highlighter/source.min.css.gz',
```
Delete also the moved CSS files.